src/lib/flow/sol-flow-buildopts.h.in

```diff
@@ -38,4 +38,6 @@
 @SOL_BUILD_FLOW_MODULE_RESOLVER_CONFFILE@
 #ifdef SOL_BUILD_FLOW_MODULE_RESOLVER_CONFFILE
 extern struct sol_flow_resolver *sol_flow_resolver_conffile;
+#else
```

Comment:
I think we should always define the builtin, not only if conffile is not defined. But @cmarcelo may have another opinion about it.